.github/workflows/pull-request.yaml

```diff
@@ -97,12 +97,12 @@ jobs:
         go-version: '1.13'
     - id: gofmt
       run: |
-        GOFMT_OUT=$(gofmt -l ./backend/src)
+        GOFMT_OUT=$(gofmt -l ./backend/api-service)
```

Comment:
Should this be more generic so that it covers any/all services in the `backend/` directory?

Reply:
Yep - that makes sense. Changed.